csunplugged/templates/dev/index.html

```diff
@@ -7,70 +7,95 @@
 {% block content %}
 
   <h1>Topics</h1>
-  <ul>
-    {% for topic in topics %}
-      <li>Topic: <a href="{% url 'topics:topic' topic.slug %}">{{ topic }}</a></li>
-      <ul>
-        {% for unit_plan in topic.unit_plans %}
-        <li>Unit Plan: <a href="{% url 'topics:unit_plan' topic.slug unit_plan.slug %}">{{ unit_plan.name }}</a></li>
+
+  {% if topics.all %}
+    <ul>
+      {% for topic in topics %}
+        <li>Topic: <a href="{% url 'topics:topic' topic.slug %}">{{ topic }}</a></li>
+
+        {% if topic.unit_plans.all %}
           <ul>
-            {% for age_group,lessons in unit_plan.lessons.items %}
-              <li>Lessons ({{ age_group.0 }}-{{ age_group.1}}):</li>
-              <ul>
-                {% for lesson in lessons %}
-                  <li><a href="{% url 'topics:lesson' topic.slug unit_plan.slug lesson.slug %}">{{ lesson }}</a></li>
-                {% endfor %}
-              </ul>
+            {% for unit_plan in topic.unit_plans %}
+              <li>Unit Plan: <a href="{% url 'topics:unit_plan' topic.slug unit_plan.slug %}">{{ unit_plan.name }}</a></li>
+
+              {% if unit_plan.lessons.items %}
+                <ul>
+                  {% for age_group,lessons in unit_plan.lessons.items %}
+                    <li>Lessons ({{ age_group.0 }}-{{ age_group.1}}):</li>
```

Comment:
Regarding Tim's style comment in Slack, we should change `{{ age_group.0 }}-{{ age_group.1}}` to `{{ age_group.0 }} to {{ age_group.1}}`